Monika After Story/game/zz_calendar.rpy

```diff
@@ -3,6 +3,9 @@
 # Contains also a store named mas_calendar which includes helper functions
 # to add Events to the calendar
 
+# Format used for calendar display
+define DATE_DISPLAY_FORMAT = "\t  \t  \t  \t  \t  \t  \t  {0}\n{1}\n{2}\n{3}"
```

Comment:
Why did you take this out of `MASCalendar`? This is extremely specific to that displayable and should remain in there.